chapters/introduction.tex

```diff
@@ -68,12 +68,37 @@ \section{Some Definitions}\label{some-definitions}
 \end{definition}
 
 \begin{definition}[Flattening]\index{flattening}
-The translation of a model described in Modelica to the corresponding model described as a hybrid DAE, involving expansion of inherited base classes, parameterization of base classes, local classes
+The translation of a model described in Modelica to the corresponding model described as a hybrid DAE (see \cref{modelica-dae-representation}), involving expansion of inherited base classes, parameterization of base classes, local classes
 and components, and generation of connection equations from \lstinline!connect!-equations (basically, mapping the hierarchical structure of a model into a set of differential, algebraic and discrete equations together with the corresponding variable declarations and function definitions from the model).
 \end{definition}
 
+\begin{definition}[Initialization]\index{initialization}
+Simulation starts with solving the initialization problem at the starting time, resulting in values for all variables that are consistent with the result of the flattening.
+\end{definition}
+
+\begin{definition}[Transient analysis]\index{transient analysis}\index{initial value problem!see{transient analysis}}
+Starting from the result of the initialization problem, the model is simulated forward in time.
+This uses numerical methods for handling the hybrid DAE, resulting in solution trajectories for the model's variables, i.e., the value of the variables as a function of time.
+\end{definition}
```

Comment:
This seems like the right place to have the document's only `\firstuse{initial value problem}`, allowing us to refer to this definition from the appendix if needed.

Reply:
I have it in the non-normative text that is now just below this; I think that suffices.

Reply:
Agreed.  Resolving.